scripts/train/benchmarking/submit_benchmarks.py

```diff
@@ -305,9 +346,23 @@ def mod_parameters(parameters: Dict[str, Any],
     if fsdp_config_activation_checkpointing is not None:
         parameters['fsdp_config'][
             'activation_checkpointing'] = fsdp_config_activation_checkpointing
-
-    parameters['fsdp_config']['activation_checkpointing_reentrant'] = False
-    parameters['fsdp_config']['limit_all_gathers'] = True
+    if fsdp_config_shard_strategy is not None:
+        parameters['fsdp_config'][
+            'sharding_strategy'] = fsdp_config_shard_strategy
+    if fsdp_config_limit_all_gathers is not None:
+        parameters['fsdp_config'][
+            'limit_all_gathers'] = fsdp_config_limit_all_gathers
+    if fsdp_config_forward_prefetch is not None:
+        parameters['fsdp_config'][
+            'forward_prefetch'] = fsdp_config_forward_prefetch
+    if fsdp_config_backward_prefetch is not None:
+        parameters['fsdp_config'][
+            'backward_prefetch'] = fsdp_config_backward_prefetch
+    if activation_cpu_offload is not None:
+        parameters['fsdp_config'][
+            'activation_cpu_offload'] = activation_cpu_offload
+    # parameters['fsdp_config']['verbose'] = True
+    parameters['compile_config'] = {} if torch_compile else None
```

Comment:
shouldn't this incorporate the compile specific args you added?

Reply:
Did you actually use compile for any or the benchmarks?
maybe this should just be deleted.

@dakinggg should we still leave it in `train.py` in case ppl want to use it in the future?

Reply:
removing compile support in profiling script